ios/GoogleService-Info.plist

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>API_KEY</key>
+	<string>AIzaSyDIjLp-OkNyBRbgr2CtX5tDBPpKcuGpx7U</string>
+	<key>GCM_SENDER_ID</key>
+	<string>1091871250916</string>
+	<key>PLIST_VERSION</key>
+	<string>1</string>
+	<key>BUNDLE_ID</key>
+	<string>org.reactjs.native.example.met-museum-rn</string>
+	<key>PROJECT_ID</key>
+	<string>metmuseum-d6bf9</string>
+	<key>STORAGE_BUCKET</key>
+	<string>metmuseum-d6bf9.appspot.com</string>
+	<key>IS_ADS_ENABLED</key>
+	<false></false>
+	<key>IS_ANALYTICS_ENABLED</key>
+	<false></false>
+	<key>IS_APPINVITE_ENABLED</key>
+	<true></true>
+	<key>IS_GCM_ENABLED</key>
+	<true></true>
+	<key>IS_SIGNIN_ENABLED</key>
+	<true></true>
+	<key>GOOGLE_APP_ID</key>
+	<string>1:1091871250916:ios:e39284ae3c59b5b61ccff1</string>
+</dict>
+</plist>
```

Comment:
Last line 😄 

Fun fact - I don't know if you've noticed, but you've pushed an API key to the repo. However, in the case of Firebase it is allowed anyway 😄 What's more no secrets are "secret" in the mobile world, because you can always download an `.apk` and get them from the source code 😄

Reply:
Should i put it to env variables or mayby in file that is gitignored?

Reply:
That could be an additional layer of protection, but I don't think it is necessary. Even our Expensify client has that pushed to open repo, so we can leave it as it is :)